app/repositories/app_usage_event_repository.rb

```diff
@@ -1,4 +1,5 @@
 require 'database/old_record_cleanup'
+require 'cloud_controller/url_secret_obfuscator'
```

Comment:
Does this relate to the change in spec helper?

Reply:
yes!